test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#GRIDDB_HOME=${HOME}/src/griddb_nosql_4.0.0
+
+if [[ ! -d "${GRIDDB_HOME}" ]]; then
+  echo "GRIDDB_HOME environment variable not set"
+  exit 1
+fi
+
+# Start GridDB server
+export GS_HOME=${GRIDDB_HOME}
+export GS_LOG=${GRIDDB_HOME}/log
+if pgrep -x "gsserver" > /dev/null
+then
+  ${GRIDDB_HOME}/bin/gs_leavecluster -w -f -u admin/testadmin
+  ${GRIDDB_HOME}/bin/gs_stopnode -w -u admin/testadmin
+fi
+rm -rf ${GS_HOME}/data/* ${GS_LOG}/*
+sleep 1
+echo "Starting GridDB server..."
+${GRIDDB_HOME}/bin/gs_startnode -w -u admin/testadmin
+${GRIDDB_HOME}/bin/gs_joincluster -w -c ktymCluster -u admin/testadmin
```

Comment:
The cluster name and admin's password are not embedded in this file.
Please modify the script as getting them from the command line argument.

Reply:
Could you please explain more about this comment?
I use these values in the script, because the test files (ex: griddb_fdw.tcl) use this value.
If we change to get them from command arguments, it might be incorrect.

Reply:
Are admin's password and cluster name embedded in the test script?
If it is correct, please change the cluster name to "griddbfdwTestCluster".

Reply:
These pieces of information are embedded in the script.
I change the cluster name to "griddbfdwTestCluster"

Reply:
Thank you for the modification.